lib/rules/sort-amd-paths.js

```diff
@@ -50,6 +50,28 @@ module.exports = function (context) {
     }
 
     /**
+     * Determine if supplied `node` represents a function expression.
+     *
+     * @private
+     * @param  {ASTNode} node - node to test
+     * @return {Boolean} true if represents a function expression
+     */
+    function isFunctionExpression(node) {
```

Comment:
`amd-function-arity` uses its own `isFunctionExpression` as well...thats why I suggested that following API change.